crates/turborepo-lib/src/shim.rs

```diff
@@ -67,6 +73,20 @@ impl ShimArgs {
             } else if arg == "--" {
                 // If we've hit `--` we've reached the args forwarded to tasks.
                 is_forwarded_args = true;
+            // parse verbosity
+            } else if let Some(verbosity_level) = arg.strip_prefix("--verbosity=") {
+                verbosity = match verbosity_level.parse::<usize>() {
+                    Ok(level) => {
+                        if level <= 3 {
+                            level
+                        } else {
+                            0
+                        }
+                    }
+                    Err(_) => 0,
+                }
+            } else if arg == "-v" || arg == "-vv" || arg == "-vvv" {
```

Comment:
Not blocking, but we did used to support any number of v's

Reply:
Good call, will fix